helm_deploy/values-uat.yaml

```diff
@@ -23,15 +23,14 @@ securityContext:
    capabilities:
      drop:
      - ALL
-   readOnlyRootFilesystem: true
    runAsNonRoot: true
    runAsUser: 10000
    allowPrivilegeEscalation: false
 
 service:
   type: ClusterIP
   port: 80
-  internalPort: 8000
+  internalPort: 3000
```

Comment:
would this also need to be changed back to get the python version working? should we chnage the python dockerfile to use 3000 instead so it's not an issue in the future?

Reply:
Good idea!